content/docs/06-integrations/00-kubernetes.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Kubernetes"
-metaTitle: "Kubernetes with Spectro Cloud"
-metaDescription: "Kubernetes pack in Spectro Cloud"
+metaTitle: "Kubernetes in Palette"
+metaDescription: "Kubernetes pack in Palette"
```

Comment:
Let's give a more insightful description for search engine optimization purposes.

Reply:
changed to say "Kubernetes pack that deploys containers in Palette"